cfbs.json

```
@@ -1167,8 +1167,8 @@
       "tags": ["supported", "security", "compliance"],
       "repo": "https://github.com/craigcomstock/cfengine-security-hardening",
       "by": "https://github.com/craigcomstock",
-      "version": "0.0.2",
-      "commit": "2ec06ea5e78b0ad39cfde0137e0a8c25a983fae8",
+      "version": "1.0.0",
+      "commit": "f065a2d245908bc0faa874d03b6721c55f924256",
```

See this [conversation](https://github.com/cfengine/core/discussions/5138) about how `inform => "false"` attribute on a commands promise doesn't inhibit command output but rather inform messages about the promise. What I really needed was `no_output => "true"` in a `contain` body.

```suggestion
      "commit": "e3039050296ec20c7e44b3accba84c146cf6ef69",
```